airflow/contrib/hooks/spark_submit_hook.py

```diff
@@ -262,16 +317,82 @@ def _process_log(self, itr):
                 if match:
                     self._yarn_application_id = match.groups()[0]
 
-            # Pass to logging
-            self.log.info(line)
+            # if we run in standalone cluster mode and we want to track the driver status
+            # we need to extract the driver id from the logs. This allows us to poll for
+            # the status using the driver id. Also, we can kill the driver when needed.
+            if self._should_track_driver_status and not self._driver_id:
+                match_driver_id = re.search('(driver-[0-9\-]+)', line)
+                if match_driver_id:
+                    self._driver_id = match_driver_id.groups()[0]
+                    self.log.info("identified spark driver id: {}".format(self._driver_id))
+
+            self.log.debug("spark submit log: {}".format(line))
```

Comment:
Could you put that status as `self.log.info` ? I find it important to always have the spark submit statement in the logs. I know it was originally in debug in the original version, though I suggest to update that. 
(same thing for the same log in `_process_spark_status_log`)

Reply:
I dont agree as it creates a lot of noise IMO.
The printed lines are also not relevant for day to day operation.
If something goes wrong, then you can start Airflow in debug mode and those lines will show up.